standard/bridge-v1/pkg/transfer/transfer.go

```diff
@@ -108,16 +118,22 @@ func NewTransferToL1(
 	}
 
 	return &Transfer{
-		amount:        amount,
-		destAddress:   destAddress,
-		privateKey:    privateKey,
-		srcClient:     commonSetup.settlementClient,
+		amount:      amount,
+		destAddress: destAddress,
+		privateKey:  privateKey,
+		srcClient: shared.NewETHClient(
+			logger.With("component", "l1_eth_client"),
+			commonSetup.l1Client,
```

Comment:
This is a transfer **to** L1, so srcClient should use `commonSetup.settlementClient` as before. 
```suggestion
			commonSetup.settlementClient,
```